cmd/main.go

```diff
@@ -76,32 +53,13 @@ func main() {
 
 	wg := &sync.WaitGroup{} // Goroutines can add themselves to this to be waited on
 
-	controllerConfig := ControllerConfig()
-
-	if configMapName != (spec.NamespacedName{}) {
-		configMap, err := controllerConfig.KubeClient.ConfigMaps(configMapName.Namespace).Get(configMapName.Name)
-		if err != nil {
-			panic(err)
-		}
-
-		configMapData = configMap.Data
-	} else {
-		log.Printf("No ConfigMap specified. Loading default values")
-	}
-	if configMapData["namespace"] == "" { // Namespace in ConfigMap has priority over env var
-		configMapData["namespace"] = podNamespace
-	}
-	if noDatabaseAccess {
-		configMapData["enable_database_access"] = "false"
-	}
-	if noTeamsAPI {
-		configMapData["enable_teams_api"] = "false"
+	config.RestConfig, err = k8sutil.RestConfig(KubeConfigFile, OutOfCluster)
```

Comment:
besides RestConfig there are some other vars passed to the Controller
see https://github.com/zalando-incubator/postgres-operator/pull/64/files/1659a9b06a2e6c22ce752508767cd145430b9647#diff-3002d695552ec7e7ebbaaf04a21d3439R26